armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -666,7 +668,7 @@ class TestDetermineTargetComponent(AxialExpansionTestBase, unittest.TestCase):
 
     def setUp(self):
         AxialExpansionTestBase.setUp(self)
-        self.expData = ExpansionData([], None)
+        self.expData = ExpansionData([], True, True)
```

Comment:
define what the variables are 
`self.expansionData = ExpansionData(a, setFuel=True, coldHeightsToHot=True)`

Reply:
https://github.com/terrapower/armi/pull/1342/commits/73c16ed9d0a14ffbf3090e619426e6c740fc38b4